backend/Models/Calendar.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CinemaApp.Models
+{
+    
+    public class Calendar
+    {
```

Comment:
Shouldn't this model have an ID or something?

Reply:
![image](https://user-images.githubusercontent.com/65727948/89295016-027edc80-d669-11ea-8cc7-ad9fc464a06e.png)

Reply:
I just followed the schema :)